rootfs/extra/etc/nginx/njs/http/cors.js

```
@@ -0,0 +1,146 @@
+
+function header_filter(r) {
+    // do not waste CPU cycles when there are no Origin request header or no CORS configuration file present
+    if (! ('http_origin' in r.variables)
+	|| ! ('cors_config_file' in r.variables)
+	|| ! r.variables.cors_config_file
+	|| 0 === r.variables.cors_config_file.length) {
+	return;
+    }
+
+    // read and parse external configuration file
+    var fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)